tsdb/store.go

```diff
@@ -231,9 +231,21 @@ func (s *Store) loadShards() error {
 
 	s.EngineOptions.CompactionLimiter = limiter.NewFixed(lim)
 
-	// Env var to disable throughput limiter.  This will be moved to a config option in 1.5.
-	if os.Getenv("INFLUXDB_DATA_COMPACTION_THROUGHPUT") == "" {
-		s.EngineOptions.CompactionThroughputLimiter = limiter.NewRate(48*1024*1024, 48*1024*1024)
+	throughput := int(s.EngineOptions.Config.CompactThroughput)
+	burstThroughput := int(s.EngineOptions.Config.CompactBurstThroughput)
+	if throughput > 0 {
+		if burstThroughput == 0 {
+			burstThroughput = throughput
+		}
+		s.Logger.Info(
+			"Compaction throughput enabled",
+			zap.String("throughput", fmt.Sprintf("%d%%", throughput)),
+			zap.String("throughput_burst", fmt.Sprintf("%d%%", burstThroughput)),
```

Comment:
Here, it's called "throughput_burst" but elsewhere in the PR it's "burstThroughput" etc. Please select one ordering.

FWIW, I prefer the "throughput burst" ordering because it keeps the config keys "compact-throughput" and "compact-throughput-burst" together visually and lexicographically.